src/coffeeMachine/Main.java

```diff
@@ -1,7 +1,42 @@
 package coffeeMachine;
-
+import java.util.Scanner;
 public class Main {
     public static void main(String[] args) {
-        System.out.print("Hello world!");
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Write how many ml of water the coffee machine has: ");
+        int water = scanner.nextInt();
+        System.out.println ("Write how many ml of milk the coffee machine has: ");
+        int milk = scanner.nextInt();
+        System.out.println ("Write how many grams of coffee beans the coffee machine has: ");
+        int coffee = scanner.nextInt();
+        System.out.println ("Write how many cups of coffee you will need: ");
+        int cups = scanner.nextInt();
+int amount_w = water/200;
```

Comment:
In Java, it is better to add one space between an operator and operands like **water / 200**;